fiber/sentryfiber.go

```diff
@@ -51,11 +55,39 @@ func New(options Options) fiber.Handler {
 
 func (h *handler) handle(ctx *fiber.Ctx) error {
 	hub := sentry.CurrentHub().Clone()
-	scope := hub.Scope()
 
-	scope.SetRequest(convert(ctx))
+	if client := hub.Client(); client != nil {
+		client.SetSDKIdentifier(sdkIdentifier)
+	}
+
+	convertedHTTPRequest := convert(ctx)
+
+	options := []sentry.SpanOption{
+		sentry.WithOpName("http.server"),
+		sentry.ContinueFromRequest(convertedHTTPRequest),
+		sentry.WithTransactionSource(sentry.SourceRoute),
+	}
+
+	method := ctx.Method()
+
+	transaction := sentry.StartTransaction(
+		sentry.SetHubOnContext(ctx.Context(), hub),
+		fmt.Sprintf("%s %s", method, ctx.Path()),
```

Comment:
Could you verify that parameters are properly parameterized?
https://docs.gofiber.io/guide/routing/#parameters

So for e.g. `app.Get("/user/:id", func(c *fiber.Ctx)` with a request like `/user/1` we would want a transaction name of `GET /user/:id`.
If this is the case, we should add some tests. If not, and there is not always a ways to get a parameterized route, we need to change the transaction source to `SourceURL`.

Reply:
They are not, I changed it. The way to get it in fiber is by using ctx.Route().Name, but that's only filled after .Next which wouldn't work in this case.